src/woo_publications/metadata/models.py

```diff
@@ -94,3 +100,44 @@ def natural_key(self):
 
     def __str__(self):
         return self.naam
+
+
+class Organisation(models.Model):
+    uuid = models.UUIDField(_("UUID"), unique=True, default=uuid.uuid4)
+    identifier = models.URLField(
+        _("identifier"),
+        help_text=_(
+            "The unique IRI that identifies this organisation in the overheid.nl value list. "
+            "For entries that have been added manually, an identifier is generated."
+        ),
+        max_length=255,
+        unique=True,
+        editable=False,
+        default=get_default_organization_identifier,
+    )
+    naam = models.CharField(_("name"), max_length=80)
+    oorsprong = models.CharField(
+        _("origin"),
+        help_text=_(
+            "Determines where the organisation is defined and sourced from, and how "
+            "the identifier should be interpreted. If the value list is the origin, the "
+            "organisation can not be modified or deleted."
+        ),
+        choices=Origins.choices,
```

Comment:
I don't think this goes like that, there is not a single value list for organisations, but a union of multiple lists that need to be imported. The `oorsprong` should reflect this. This is pointed to from the ticket: https://github.com/GeneriekPublicatiePlatformWoo/registratie-component/issues/8#issuecomment-2325940935